ts/components/leftpane/overlay/OverlayMessageRequest.tsx

```diff
@@ -77,6 +76,7 @@ export const OverlayMessageRequest = () => {
 
           for (let index = 0; index < messageRequests.length; index++) {
             const convoId = messageRequests[index];
+            // eslint-disable-next-line no-await-in-loop
```

Comment:
I feel like we do await in a for loop often enough that we disable this rule globally? It always work better when forEach fails doing async properly

Reply:
I'd rather have to disable it explicitly when needed for now. It's easier to check PRs when there is an explicit eslint-disable to see if it is indeed needed or not.

Reply:
Fair enough. 👍